main

```diff
@@ -22,7 +22,11 @@ proc ModulesHelp { } {
 set product desimodules
 set version main
 conflict $product
-set desiconda_version 20211217-2.0.0
+if { $env(NERSC_HOST) == "perlmutter" } {
```

Comment:
This looks fine at NERSC, but what happens when $NERSC_HOST isn't set, e.g. at KPNO?  Does it pass through to the alternate desiconda version (ok), or crash (bad)?  Please test that case, and if needed update it e.g. using syntax like `if{[info exists env(NERSC_HOST)]}` like we have with $DESI_PRODUCT_ROOT:

```
if {[info exists env(DESI_PRODUCT_ROOT)]} {
    set code_root $env(DESI_PRODUCT_ROOT)/code
} else {
    set code_root /global/common/software/desi/$env(NERSC_HOST)/desiconda/$desiconda_version/code
}
```

We don't yet have 20211217-2.0.0 at KPNO, but we shouldn't arbitrarily break non-NERSC usage of desimodules either.